Userland/Services/WindowServer/Compositor.cpp

```diff
@@ -1096,7 +1097,8 @@ void Compositor::recompute_occlusions()
         });
 
         m_opaque_wallpaper_rects.clear();
-    } else {
+    }
+    {
```

Comment:
I'm not quite sure why this (massive) else branch is now always being run?

Reply:
I changed this line again, can you have a look? The idea is basically to keep drawing the background, when the fullscreen window is transparent.

Reply:
looks like it does what you're saying it does, so that's alright by me :P